research/src/sources/atlassian.json5

```diff
@@ -1,634 +1,656 @@
 {
-  "$schema": "../schemas/design-system.schema.json5",
-  "lastUpdated": "2019-08-22",
-  "name": "Atlaskit",
-  "description": "Atlassian's official UI library, built according to the Atlassian Design Guidelines.",
-  "url": "https://atlaskit.atlassian.com",
-  "by": "Atlassian",
-  "components": [
+  $schema: '../schemas/design-system.schema.json5',
```

Comment:
It seems like there was an auto-update on this file, do you think you can check if you can remove that so we can quickly identify the actual changes?

Reply:
Ah completely missed this thanks, I reverted and reapplied